src/reader/arrow_transform.ts

```diff
@@ -156,21 +157,33 @@ export class ArrowRecordBatchTableRowTransform extends Transform {
   }
 }
 
-function convertArrowValue(fieldValue: any): any {
-  if (typeof fieldValue === 'object') {
-    if (fieldValue instanceof Vector) {
-      const arr = fieldValue.toJSON();
-      return arr.map((v: any) => {
-        return {v: convertArrowValue(v)};
-      });
-    }
-    const tableRow: TableRow = {f: []};
+function convertArrowValue(fieldValue: any, type: DataType): any {
+  if (fieldValue === null) {
+    return null;
```

Comment:
Do we also need to care about it being undefined?

Reply:
that value is coming from the BigQuery backend, so undefined is not a possible value in that scenario.